nomad/structs/network_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"testing"
 
 	"github.com/hashicorp/nomad/ci"
+	"github.com/shoenig/test/must"
```

Comment:
If the plan is to backport the bug fix we unfortunately can't use this yet 😢

Reply:
Doh, of course. I forgot `main` isn' 1.3.2. Thanks!